posts/_markdown/2021-member-announcement.md

```diff
@@ -0,0 +1,38 @@
+
+Today, Unitary Fund is proud to announce our new corporate member program. We thank these organizations for their support and look forward to their further contributions in the coming year.
+
+The corporate member program contributes to Unitary Fund’s work on a recurring, annual basis, and consists of two tiers: cCore members and supporting members. core members' contributions start at $100k annually and supporting member’s contributions start at $25k annually. Additionally, core members are a part of the Unitary Fund’s Strategic Board which reviews strategy, budget, impact, and recommendation’s quarterly.
+
+## Unitary Fund Core Members: IBM and Accenture
+
+![](../images/core-members-2021.png "Core members IBM and Accenture")
+
+“Helping talented individuals with the opportunity to advance their innovative ideas is a core part to growing the quantum ecosystem.  We are especially excited to enable underrepresented entrepreneurs through the Unitary Fund as part of our Corporate Citizenship efforts,” said Carl Dukatz, Accenture Quantum Program Lead.  “Additionally, Accenture has a long history of contributing to open-source and is glad to work with the Unitary Fund to create the important foundations for quantum.”
+
+The Unitary Fund awards $4,000 grants to individuals and teams pursuing projects that leverage quantum technologies to benefit humanity. Projects could be open source quantum software, educational materials and workshops, a new quantum sensor prototype, or much more. The program is unique in its inclusivity (no credentials are required to apply) and low friction (awards are no-strings-attached).
+
+## Unitary Fund Supporting Members: Xanadu, IonQ, BCG, Pasqal, IQM, and DoraHacks
+
+![](../images/supporting-members-2021.png "Unitary Fund Supporting Members: Xanadu, IonQ, BCG, Pasqal, IQM, and DoraHacks")
+
+"It was important for Pasqal to reaffirm its support to open-source initiatives in quantum software by becoming a sponsor of Unitary Fund” says Loic Henreit, CTO at France-based quantum computing startup Pasqal. “We are thrilled to join the efforts led by Unitary Fund to build a solid ecosystem in this field."
+
+"We are delighted to support the Unitary Fund's mission. They are doing first-class work building out a quantum ecosystem for everyone. You can see the Unitary Fund's impact everywhere you look," said Nathan Killoran, Head of Software, Algorithms, & Quantum Machine Learning at Xanadu, a photonic quantum computer company in Canada.
+
+Quantum computers have the potential to be revolutionary for many industries with its capability to develop efficient solutions for computational problems beyond the scope of a conventional machine and uncover unimaginable new possibilities.
```

Comment:
```suggestion
```

Reply:
If the suggestion to cut is not accepted, "its" should eventually be changed to "their"

Reply:
I'd support this cut